content/_data/logo/jenkins-x.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: 'Jenkinx-X'
+url: '/images/logos/jenkins-x/jenkins-x.png'
+url_256: '/images/logos/jenkins-x/jenkins-x-256.png'
+vector: '/images/logos/jenkins-x/jenkins-x.svg'
+credit: 'Joshua'
```

Comment:
It would be better to have a full name and a link to his profile (GitHub/Twitter)